content/integrations/servicenow.md

```diff
@@ -2,6 +2,7 @@
 title: Datadog-ServiceNow Integration
 integration_title: ServiceNow
 kind: integration
+description: "Have your Datadog alerts automatically generate and update tickets."
```

Comment:
Have your Datadog alerts automatically generate and update tickets in ServiceNow.

Reply:
👍